aptos-node/src/tests.rs

```diff
@@ -51,3 +51,57 @@ fn test_mutual_authentication_validators() {
 fn test_aptos_vm_does_not_have_test_natives() {
     aptos_vm::natives::assert_no_test_natives(crate::utils::ERROR_MSG_BAD_FEATURE_FLAGS)
 }
+
+#[test]
+fn test_create_single_node_test_config() {
+    // create a test config override and merge it with the default config
+    // this will get cleaned up by the tempdir when it goes out of scope
+    let test_dir = aptos_temppath::TempPath::new().as_ref().to_path_buf();
+    fs::DirBuilder::new()
+        .recursive(true)
+        .create(&test_dir)
+        .expect("Failed to create test_dir");
+    let config_path = None;
```

Comment:
Why do you need a variable for this? 😄 Just pass `None` into `create_single_node_test_config`

Reply:
🤦 thanks for catching this lol